xbmc/filesystem/IFileTypes.h

```diff
@@ -14,39 +14,44 @@ namespace XFILE
 {
 
 /* indicate that caller can handle truncated reads, where function returns before entire buffer has been filled */
-  static const unsigned int READ_TRUNCATED = 0x01;
+static const unsigned int READ_TRUNCATED = 0x01;
 
-/* indicate that that caller support read in the minimum defined chunk size, this disables internal cache then */
-  static const unsigned int READ_CHUNKED = 0x02;
+/*
+ * Indicate that caller support read in the minimum defined chunk size, this disables internal cache then.
+ * This flag is deprecated, instead use READ_NO_CACHE to disable FileCache and READ_NO_BUFFER to disable StreamBuffer
+ * On the contrary to explicitly indicate that the file has audio/video content (suitable for caching), use
+ * the READ_AUDIO_VIDEO flag.
+ */
+static const unsigned int READ_CHUNKED = 0x02;
```

Comment:
I'd either remove it all together or at least comment it out.

Reply:
Yes, I agree, maybe this flag should be completely removed and update `READ_AUDIO_VIDEO` flag comment.

Reply:
Removed and updated the description of READ_AUDIO_VIDEO flag.